HPCC-13084 DOCS:UPdates to SystemAdmin Doc

g-pan opened the pull request:
FIX HPCC-13084 DOCS:UPdates to SystemAdmin Doc
UPdates and additions to System Administrators Doc.

Signed-off-by: G Panagiotatos greg.panagiotatos@lexisnexis.com

@JamesDeFabia please review

g-pan committed:
HPCC-13084 DOCS:UPdates to SystemAdmin Doc

FIX HPCC-13084 DOCS:UPdates to SystemAdmin Doc
UPdates and additions to System Administrators Doc.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

g-pan committed:
HPCC-13084 DOCS:UPdates to SystemAdmin Doc

FIX HPCC-13084 DOCS:UPdates to SystemAdmin Doc
UPdates and additions to System Administrators Doc.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

g-pan committed:
HPCC-13084 DOCS:UPdates to SystemAdmin Doc

FIX HPCC-13084 DOCS:UPdates to SystemAdmin Doc
UPdates and additions to System Administrators Doc.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

JamesDeFabia commented on docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml:
```diff
@@ -825,7 +978,7 @@
       <para>The default envrionment.conf:</para>
 
-      <para><programlisting>## HPCC Systems default environment configuration file 
+      <para><programlisting>## HPCC Systems<superscript>®</superscript> default environment configuration file 
```
You cannot alter the display of the conf file's test here unless you have also edited the default environment.conf file

JamesDeFabia commented on docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml:
```diff
@@ -1850,7 +1345,7 @@ sudo cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/environment.
             <para>Open the ECL IDE (Start <emphasis
             role="bold">&gt;&gt;</emphasis> All Programs <emphasis
-            role="bold">&gt;&gt;</emphasis> HPCC Systems <emphasis
+            role="bold">&gt;&gt;</emphasis> HPCC Systems<superscript>®</superscript> <emphasis
```
Does the ® symbol appear on your Windows Start Menu option? It doesn't on mine.

JamesDeFabia commented on docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml:
```diff
@@ -2547,7 +2042,7 @@ OUTPUT(ValidWords)
 
     <para>Start <emphasis role="bold">&gt;&gt;</emphasis> All Programs
-    <emphasis role="bold">&gt;&gt;</emphasis> HPCC Systems <emphasis
+    <emphasis role="bold">&gt;&gt;</emphasis> HPCC Systems<superscript>®</superscript> <emphasis
```
Does the ® symbol appear on your Windows Start Menu option? It doesn't on mine

g-pan committed:
HPCC-13084 DOCS:UPdates to SystemAdmin Doc

FIX HPCC-13084 DOCS:UPdates to SystemAdmin Doc
UPdates and additions to System Administrators Doc.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

JamesDeFabia commented:
Looks good to merge now
